### PR TITLE
Add Configuration as Code Groovy extension plugin

### PR DESCRIPTION
**Pull request by szandala**
# Description
Adding permission for releasing new plugin:
https://github.com/jenkinsci/configuration-as-code-groovy-plugin
for user szandala
Plugins is hosted since 
https://issues.jenkins-ci.org/browse/HOSTING-714

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

File: permissions/plugin-configuration-as-code-groovy.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "configuration-as-code-groovy"
+github: "jenkinsci/configuration-as-code-groovy-plugin"
+paths:
+- "org/jenkins-ci/plugins/configuration-as-code-groovy"
```
Comment (slide) [inline permissions/plugin-configuration-as-code-groovy.yml]: Is this path needed? Your groupId is "io.jenkins"

Comment (szandala) [reply permissions/plugin-configuration-as-code-groovy.yml]: Please check now. I do not get this "paths" part

**Comment by slide** [comment]
The path is basically `<groupId>` from pom.xml replacing `.` with `/` then `<artifactId>`. So, since your `groupId` is `io.jenkins.plugins` (https://github.com/jenkinsci/configuration-as-code-groovy-plugin/blob/master/pom.xml#L12) and your `artifactId` is `configuration-as-code-groovy` (https://github.com/jenkinsci/configuration-as-code-groovy-plugin/blob/master/pom.xml#L13) you would have:

`io/jenkins/plugins/configuration-as-code-groovy-plugin`

**Comment by szandala** [comment]
> The path is basically `<groupId>` from pom.xml replacing `.` with `/` then `<artifactId>`. So, since your `groupId` is `io.jenkins.plugins` (https://github.com/jenkinsci/configuration-as-code-groovy-plugin/blob/master/pom.xml#L12) and your `artifactId` is `configuration-as-code-groovy` (https://github.com/jenkinsci/configuration-as-code-groovy-plugin/blob/master/pom.xml#L13) you would have:
> 
> `io/jenkins/plugins/configuration-as-code-groovy`

Sooo, now it should be fine, isn't it?

**Comment by slide** [comment]
Yes, just wanted to make sure you understood :-)

**Comment by szandala** [comment]
Thanks :D

**Pull request merged**